### PR TITLE
Fixes Issue #14: Add Github Action to check with flake8

### LINKED ISSUE #14: Add Github Action to check with flake8

### PR DESCRIPTION
**Pull request by OBITORASU**
## Description
Added a GitHub Actions workflow based on a market template which checks the codebase with flake8.

Fixes #14

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout source repo
+      uses: actions/checkout@v2
+    - name: Install Python 
+      uses: actions/setup-python@v2
+      with:
```
Comment (huenique) [inline .github/workflows/lint.yml:11]: @OBITORASU please add `max-line-length: "88"` as per Black's [default](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length) maximum line length

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+          python-version: 3.8
```
Comment (huenique) [inline .github/workflows/lint.yml:12]: Replace `python-version: 3.8` with `python-version: 3.9`. Dayong isn't compatible with python 3.8 or earlier versions

**Comment by OBITORASU** [comment]
Fixed!

**Review by huenique — APPROVED** [review]
LGTM 👍 oh, and thanks!

**Pull request merged**